queue_services/entity-emailer/src/entity_emailer/email_processors/correction_notification.py

```diff
@@ -45,6 +47,10 @@ def _get_pdfs(
         'Authorization': f'Bearer {token}'
     }
     legal_type = business.get('legalType', None)
+    is_cp_special_resolution = (
+        legal_type == Business.LegalTypes.COOP.value and
+        filing.filing_json['filing']['correction'].get('correctedFilingType') == 'specialResolution'
+    )
```

Comment:
so do you need to handle the correction scenario where you do a correction on the original SR correction like you did for the filer?

Reply:
yes, thanks for point out, I just added a is_special_resolution_correction helper function to trace original sr filing.